Features/update 3.21

rajatshuvro opened the pull request.

rroy1-illumina committed:
adding versioned doc and sidebar, updating for 3.21

rroy1-illumina committed:
update test script

rroy1-illumina committed:
removing uga references

rroy1-illumina committed:
Merge branch 'master' into features/update_3.21

rroy1-illumina committed:
updating 3.21 docs

rajatshuvro commented on static/files/TestNirvana.sh:
```diff
@@ -56,58 +57,33 @@ popd () {
     command popd "$@" > /dev/null
 }
 
```
remove commented code.

rajatshuvro commented on static/files/TestNirvana.sh:
```diff
@@ -4,15 +4,16 @@
 # folders have been downloaded into the NIRVANA_ROOT folder.
 
-# In addition to downloading the Nirvana data files, make sure you have .NET Core 2.0
+# In addition to downloading the Nirvana data files, make sure you have .NET 6.0
```
Move this file to a non static location from 3.21

rroy1-illumina committed:
Merge branch 'master' into features/update_3.21

Fahd-Siddiqui commented on versioned_docs/version-3.21/introduction/getting-started.md:
```diff
@@ -0,0 +1,215 @@
+
+### GitHub Release Notes
+
+Alternatively, you can grab the latest binaries from our [GitHub Releases](https://github.com/Illumina/Nirvana/releases) page:
```
Should we change the github link to git.illumina link since 3.21 will reside on git.illumina?

Fahd-Siddiqui commented on versioned_docs/version-3.21/introduction/getting-started.md:
```diff
@@ -0,0 +1,215 @@
+
+## Quick Start
+
+If you want to get started right away, we've created [a script](https://illumina.github.io/NirvanaDocumentation/files/TestNirvana.sh) that downloads Nirvana, compiles it, and starts annotating a test file:
```
I believe we had decided to link (and move the script) to versioned address?

fsiddiqui-illumina committed:
rollback TestNirvana.sh file
added TestNirvana.sh to dynamic location
minor updates to getting-started.md instructions

fsiddiqui-illumina committed:
updated the wording

rroy1-illumina committed:
adding build zip in the command line

rroy1-illumina committed:
updating top directory to be current dir

Commit:
fixing the test script

rajatshuvro reviewed (commented):
looks good. I can't approve though.

Commit:
pushing to versioned doc
